config/config_schema.json

```diff
@@ -92,7 +92,7 @@
         },
         "mir_list":{
             "type": "array",
-            "default": ["miRNA", "miRNA_primary_transcript", "isomirs"]
+            "default": ["iso_mirna", "pri_mir"]
```

Comment:
I have to use `mirna` instead of `mir` because otherwise, the script `merge_tables.R` takes the `iso` part of `isomir` as part of the library name when merging the `mir` tables  (idk why)